Lib9c/Action/DPoS/Model/Validator.cs

```diff
@@ -42,7 +42,7 @@ public Validator(IValue serialized)
         // TODO: Better structure
         // This hard coding will cause some problems when it's modified
         // May be it would be better to be serialized
-        public static FungibleAssetValue MinSelfDelegation => Asset.ConsensusToken * 1;
+        public static FungibleAssetValue MinSelfDelegation => Asset.ConsensusFromGovernance(1);
```

Comment:
Not sure, if `MinSelfDelegation` have to be defined within `ConsensusToken` or `GovernanceToken`. 🤔
Since it's defined on the app side, may be current notation is right.

Reply:
https://github.com/planetarium/lib9c/pull/2521/commits/d0ba581b3a66485d8e1f4c8214dcc52f53d5fec0 amended.